src/button/__tests__/button.test.js

```diff
@@ -65,19 +65,4 @@ describe('Button Component', () => {
       target: 'foo',
     });
   });
-
-  test('internalOnClick prevents external onClick while loading', () => {
```

Comment:
Not sure what this test is about, but should we still keep it as it tests an edge case?